### PR TITLE
file: Update to 5.34

### PR DESCRIPTION
**Pull request by neheb**
Get rid of autoreconf to speed up build.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @ratkaj 
Compile tested: ipq806x

### CODE REVIEW

File: libs/file/Makefile [hunk 1]
```diff
@@ -8,20 +8,19 @@
...
 	http://pkgs.fedoraproject.org/lookaside/pkgs/file/ \
 	http://download.openpkg.org/components/cache/file/ \
 	ftp://ftp.astron.com/pub/file/
-PKG_HASH:=1c52c8c3d271cd898d5511c36a68059cda94036111ab293f01f83c3525b737c6
+PKG_HASH:=f15a50dbbfa83fec0bd1161e8e191b092ec832720e30cd14536e044ac623b20a
 
 PKG_LICENSE:=BSD-2c
 PKG_LICENSE_FILES:=COPYING
 
-PKG_FIXUP:=autoreconf
```
[the remark this answers is not in the document]
Comment (neheb) [reply libs/file/Makefile]: I assume it's due to a previous version patching configure.ac to fix build problems and it being forgotten upon version bump.

[the remark this answers is not in the document]
Comment (pprindeville) [reply libs/file/Makefile]: Can you confirm that the same `configure` (et al) files are generated either way?

[the remark this answers is not in the document]
Comment (neheb) [reply libs/file/Makefile]: No. Different autoconf versions are used on my system. There are slight differences as a result.

**Comment by neheb** [comment]
Force pushed a URL fix.

File: libs/file/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (pprindeville) [reply libs/file/Makefile]: Right, and I expected that.  I'm asking about substantive differences.

[the remark this answers is not in the document]
Comment (pprindeville) [reply libs/file/Makefile]: Right, and I expected that. I'm asking about substantive differences.

[the remark this answers is not in the document]
Comment (neheb) [reply libs/file/Makefile]: Nope. Compiled size is the same with and without.

**Comment by neheb** [comment]
Force pushed the same commit after a git rebase. Travis was complaining something about a merge request.

File: libs/file/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (pprindeville) [reply libs/file/Makefile]: Wasn't thinking about the size... more about the defaults in a shipped `./configure` file versus one re-synthesized via `autoreconf`.

File: libs/file/Makefile [hunk 2]
```diff
@@ -8,28 +8,27 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=file
-PKG_VERSION:=5.33
-PKG_RELEASE:=2
+PKG_VERSION:=5.34
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://sources.lede-project.org/ \
```
Comment (diizzyy) [inline libs/file/Makefile]: Remove sources as PKG_SOURCE_URL, it's already in the fallback mechanism
Fedora URL --> HTTPS

**Comment by neheb** [comment]
I think this is what you meant?

**Comment by diizzyy** [comment]
You're correct :)

**Pull request merged**